packages/libraries/yoga/tests/yoga.spec.ts

```diff
@@ -420,36 +420,30 @@ test('reports usage with response cache', async ({ expect }) => {
       }),
     ],
   });
-  await new Promise<void>((resolve, reject) => {
```

Comment:
Previous implementation was not consistently resolving in time. I simplified it by making it serial awaits, and added a wait after them to trigger the event loop. This seems much more consistent.